notification-mute@jgillula/files/notification-mute@jgillula/metadata.json

```diff
@@ -1,6 +1,5 @@
 {
-    "description": "Click to mute notifications, click again to unmute",
     "uuid": "notification-mute@jgillula",
     "name": "Notification Mute",
-    "max-instances": "-1"
+    "description": "Click to mute notifications, click again to unmute",
```

Comment:
Hello, @jgillula.

This is invalid JSON. Remove the last comma.

Reply:
Fixed! Thanks for catching that--I'm an idiot.